src/engraving/libmscore/ambitus.cpp

```diff
@@ -376,13 +371,13 @@ mu::PointF Ambitus::pagePos() const
 
 void Ambitus::normalize()
 {
-    if (_topPitch < _bottomPitch) {
-        int temp    = _topPitch;
-        _topPitch   = _bottomPitch;
-        _bottomPitch= temp;
-        temp        = _topTpc;
-        _topTpc     = _bottomTpc;
-        _bottomTpc  = temp;
+    if (m_topPitch < m_bottomPitch) {
+        int temp    = m_topPitch;
+        m_topPitch   = m_bottomPitch;
+        m_bottomPitch= temp;
+        temp        = m_topTpc;
+        m_topTpc     = m_bottomTpc;
+        m_bottomTpc  = temp;
```

Comment:
And this can be cleaned up a lot by using std::swap... but that too can be done later.